.travis.yml

```diff
@@ -9,6 +9,7 @@ rvm:
   - 2.1.10
   - 2.2.5
```

Comment:
Sure, I can do that in an additional commit or PR if it's acceptable.  I wanted to only add 2.4 in this PR.  I'm ok either way.  It's easy enough to change the description.